docs/en/install.md

```diff
@@ -1,202 +1,215 @@
 ```shell
 conda install pytorch torchvision -c pytorch
 ```
 
-```{note}
-Make sure that your compilation CUDA version and runtime CUDA version match.
+```{warning}
+This command will automatically install the latest version PyTorch and cudatoolkit, please check whether they matches your environment.
 ```
 
-You can check the supported CUDA version for precompiled packages on the [PyTorch website](https://pytorch.org/).
-
-`E.g.1` If you have CUDA 10.2 installed under `/usr/local/cuda` and would like to install PyTorch 1.8.0,
-you need to install the prebuilt PyTorch with CUDA 10.2.
+On CPU platforms:
 
 ```shell
-conda install pytorch==1.8.0 torchvision==0.9.0 cudatoolkit=10.2 -c pytorch
+conda install pytorch torchvision cpuonly -c pytorch
 ```
 
-`E.g.2` If you have CUDA 9.2 installed under `/usr/local/cuda` and would like to install PyTorch 1.7.0.,
-you need to install the prebuilt PyTorch with CUDA 9.2.
+# Installation
+
+We recommend that users follow our best practices to install MMPose. However, the whole process is highly customizable. See [Customize Installation](#customize-installation) section for more information.
+
+## Best Practices
+
+**Step 0.** Install [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
 
 ```shell
-conda install pytorch==1.7.0 torchvision==0.8.0 cudatoolkit=9.2 -c pytorch
+pip install -U openmim
+mim install mmcv-full
 ```
 
-If you build PyTorch from source instead of installing the pre-built package, you can use more CUDA versions such as 9.0.
+**Step 1.** Install MMPose.
 
-## Install MMPose
+Case a: If you develop and run mmpose directly, install it from source:
+
+```shell
+git clone https://github.com/open-mmlab/mmpose.git
+cd mmpose
+pip install -v -e .
+# "-v" means verbose, or more output
+# "-e" means installing a project in editable mode,
+# thus any local modifications made to the code will take effect without reinstallation.
+```
 
-a. Install mmcv, we recommend you to install the pre-built mmcv as below.
+Case b: If you use mmpose as a dependency or third-party package, install it with pip:
 
 ```shell
-# pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/{cu_version}/{torch_version}/index.html
-pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.9.0/index.html
-# We can ignore the micro version of PyTorch
-pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.9/index.html
+pip install mmpose
 ```
 
-mmcv-full is only compiled on PyTorch 1.x.0 because the compatibility usually holds between 1.x.0 and 1.x.1. If your PyTorch version is 1.x.1, you can install mmcv-full compiled with PyTorch 1.x.0 and it usually works well.
+## Verify the installation
 
-See [here](https://github.com/open-mmlab/mmcv#installation) for different versions of MMCV compatible to different PyTorch and CUDA versions.
+To verify whether MMPose is installed correctly, we provide some sample codes to run an inference demo.
```

Comment:
To verify that MMPose is installed correctly, you can run an inference demo with the following steps.
```suggestion
To verify whether MMPose is installed correctly, we provide some sample codes to run an inference demo.
```